sdk/core/azure-core/azure/core/_pipeline_client.py

```diff
@@ -102,20 +113,34 @@ def _build_pipeline(self, config, **kwargs): # pylint: disable=no-self-use
         policies = kwargs.get('policies')
 
         if policies is None:  # [] is a valid policy list
+            per_call_policies = kwargs.get('per_call_policies', [])
+            per_retry_policies = kwargs.get('per_retry_policies', [])
             policies = [
                 RequestIdPolicy(**kwargs),
                 config.headers_policy,
                 config.user_agent_policy,
                 config.proxy_policy,
-                ContentDecodePolicy(**kwargs),
-                config.redirect_policy,
-                config.retry_policy,
-                config.authentication_policy,
-                config.custom_hook_policy,
-                config.logging_policy,
-                DistributedTracingPolicy(**kwargs),
-                config.http_logging_policy or HttpLoggingPolicy(**kwargs)
+                ContentDecodePolicy(**kwargs)
             ]
+            if isinstance(per_call_policies, Iterable):
+                for policy in per_call_policies:
+                    policies.append(policy)
+            else:
+                policies.append(per_call_policies)
+
+            policies = policies + [config.redirect_policy,
+                               config.retry_policy,
+                               config.authentication_policy,
+                               config.custom_hook_policy]
```

Comment:
This causes a new `list` object to be created and assigned to `policies`. [`list.extend(iterable)`](https://docs.python.org/3/tutorial/datastructures.html#more-on-lists) is a better choice as it modifies `policies` directly.